### ISSUE TITLE
Plunker Search-box demo

### ISSUE THREAD

**Issue by jcastro3** [angular-ui/angular-google-maps#1227]
TypeError: Cannot read property 'SearchBox' of undefined

It seems the Google Places library is missing form the demo
    <script src='http://maps.googleapis.com/maps/api/js?libraries=places&sensor=false&language=en&v=3.17'></script>

**Comment by kandizzy** [comment]
@jcastro3 you are right. [here's one that works](http://plnkr.co/edit/Qsm9KgByM24i1QhLbTfK?p=preview) in the meantime before that's fixed.

**Comment by kandizzy** [comment]
@nmccready I made this fix today in my local website branch, but it seems they are already there. the PR said the branches were equal.

**Comment by cthrax** [comment]
@kandizzy that likely means a "grunt gh-pages" is necessary to publish the changes to the website. Since you have collaborator status, you ought to be able to run that fine.

**Comment by cthrax** [comment]
Make sure that you do a "grunt" before running that though, to get the current build in the right place. (probably ought to look at just making that a dependency of the gh-pages task)

**Comment by kandizzy** [comment]
hi @cthrax I got the plunkr fixed and made one other small fix on the search-box documentation page. Although the plunkr works, the map on the page is still not finding the places library. I will take a look at that tomorrow.
